src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int amountOfBuy = 0;
+    private int amountOfSupply = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readFromFile(fromFileName)));
```

Comment:
Split this line into 3 ones to make it easy to read.
String data = (readFromFile(fromFileName);
String report = ...